helm/cert-manager-app/values.yaml

```diff
@@ -131,6 +131,12 @@ crds:
   # be managed by another method.
   install: true
 
+  # crd.installationJob
+  installationJob:
```

Comment:
IMO the whole `crds` map is already configuring the CRD installation job. Not really a blocker from my side, but I would omit the additional `installationJob` map and put `ttlSecondsAfterFinished` directly under the `crds` map.

Reply:
Updated 👍🏻